ecommerce/frontend/src/components/ProductCard.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react'
+import { BsPlusCircle } from 'react-icons/bs'
+import { Button, Rate } from 'antd'
+import { Card } from 'antd'
+
+const { Meta } = Card
+
+export const ProductCard = ({ Title, Price, Rating = 0 }) => {
```

Comment:
also since price will be a singular decimal value like "49.99" I would suggest asking for a currency parameter since it is possible to have different currencies.
then you would show {price} {currency} in the render

Reply:
Yes, sending Product object will be better and easier. I created dummy product object : You can also use them while testing 
`const products = [
    {
        title: 'Yellow Flowers',
        price: '14.99',
        currency: '₺',
        productId: 1,
        rating: '2.5',
        width: 300,
        imageUrl:
            'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcS9I14rV_Z52uoBWHbruHsl84HI13mv66EW8A&usqp=CAU',
    },
    {
        title: 'Blue Toy Car',
        price: '23.99',
        currency: '€',
        productId: 2,
        rating: '3.5',
        width: 400,
        imageUrl: 'https://i.ebayimg.com/images/g/2GgAAOSw~PVcpcby/s-l400.jpg',
    },
]`
For now, currency operations are not done in frontend, FE accepts the value and the currency as input and just renders them.

Reply:
thanks a lot!